java/ql/src/Security/CWE/CWE-079/AndroidWebViewSettingsEnabledJavaScript.qhelp

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE qhelp PUBLIC
+ "-//Semmle//qhelp//EN"
+ "qhelp.dtd">
+<qhelp>
+  <overview>
+<p>
+  Enabling JavaScript in an Android WebView allows for the running of JavaScript
+code in the context of the running application. This opens the possibility for a
+man-in-the-middle attack, where the attacker can inject arbitrary JavaScript.
```

Comment:
The attack vector isn't only a MitM attack: if the application loads untrusted content in the WebView, attackers could provide malicious JavaScript to be executed as well.